lib/daffy_lib/validators/string_validator.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
```

Comment:
Arguably this should be its own gem as well since it's used throughout. 

For future consideration.